src/ucp/proto/proto_debug.h

```diff
@@ -172,4 +168,12 @@ void ucp_proto_select_elem_trace(ucp_worker_h worker,
                                  ucp_proto_select_elem_t *select_elem);
 
 
+void ucp_proto_select_write_info(
```

Comment:
`ucp_proto_debug_...`

Reply:
Let's make a common decision for all such cases in [that conversation](https://github.com/openucx/ucx/pull/10051#discussion_r1734616108)